README.md

```diff
@@ -71,65 +72,96 @@ Provides the following HTTP methods for `/load-test` endpoint:
+
+```shell
+$ helm repo add kangal https://hellofresh.github.io/kangal
+$ helm install --set environment=dev kangal kangal/kangal
+```
+
+That's it, Kangal should be installed, check if is all correct by running:
+
+```shell
+$ kubectl get pods
+NAME                                 READY   STATUS    RESTARTS   AGE
+kangal-controller-588677b854-r9qcs   1/1     Running   0          44s
+kangal-openapi-ui-7c5dd8997c-jj4mk   1/1     Running   0          44s
+kangal-openapi-ui-7c5dd8997c-vgm8c   1/1     Running   0          44s
+kangal-proxy-7d95c9d65-6t44b         1/1     Running   0          44s
+kangal-proxy-7d95c9d65-75dv4         1/1     Running   0          44s
+
+$ kubectl get crd
+NAME                                          CREATED AT
+loadtests.kangal.hellofresh.com               2019-11-15T10:59:05Z
+```
+
+For more information about the Helm Chart check [charts/kangal/README.md](charts/kangal/README.md).
+
+### Creating first LoadTest
+To run a LoadTest you first need to find Kangal proxy endpoint. Use this command:
 
-To start contributing, please check [CONTRIBUTING](CONTRIBUTING.md).
+```shell
+$ kubectl get ingress
+NAME                HOSTS                  ADDRESS     PORTS   AGE
+kangal-openapi-ui   kangal-openapi.local   localhost   80      5m48s
+kangal-proxy        kangal-proxy.local     localhost   80      5m48s
+```
+
+> This is assuming you have a properly configured [Ingress Controller](https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/). If it is not the case you can use [Port Forwarding](https://kubernetes.io/docs/tasks/access-application-cluster/port-forward-access-application-cluster/).
+
+With this information, you are now able to do a request to create the first load test.
+Let's start by downloading an example JMeter test and POST it to Kangal proxy.
+
+```shell
+$ curl -s -O https://raw.githubusercontent.com/hellofresh/kangal/master/examples/constant_load.jmx
+$ curl \
+    -H "Host: kangal-proxy.local" \
+    -F "distributedPods=1" \
+    -F "testFile=@constant_load.jmx" \
+    -F "type=JMeter" \
+    -F "overwrite=true" \
+    http://localhost:80/load-test
+{"type":"JMeter","distributedPods":1,"loadtestName":"loadtest-dunking-hedgehog","phase":"creating","hasEnvVars":false,"hasTestData":false}
+```
+
+Your first load test was created successfully, in this example with the name `loadtest-dunking-hedgehog`.
+
+Kangal Controller will automatically create a namespace for your load test and deploy the backend (in this case JMeter), check that by running:
+
+```
+$ kubectl get namespaces
+NAME                        STATUS   AGE
+...
+loadtest-dunking-hedgehog   Active   3s
+```
+
+And you can check if the Pods started correctly using:
+
+```
+$ kubectl get pods --namespace=loadtest-dunking-hedgehog
+NAME                    READY   STATUS    RESTARTS   AGE
+loadtest-master-f6xpb   1/1     Running   0          18s
+loadtest-worker-000     1/1     Running   0          22s
+```
```

Comment:
I think after that we should say also about how to check the status of LoadTest and point to openapi.json schema how to make that kind of requests.

Reply:
Good point, I will do that

Reply:
Done!